lte/gateway/docker/services/enodebd/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:20.04 as enodebd
```

Comment:
Can the base image be a variable passed as well? This would help the upgrade process in the future.